pkgs/top-level/darwin-packages.nix

```diff
@@ -15,6 +15,7 @@ in
   binutils-unwrapped = callPackage ../os-specific/darwin/binutils {
     inherit (darwin) cctools;
     inherit (pkgs) binutils-unwrapped;
+    inherit (pkgs.llvmPackages_5) llvm;
```

Comment:
I'm not sure using an explicit version here is a good idea, it's one more thing to keep in sync when updating the default version.